lib/download.js

```diff
@@ -86,13 +89,28 @@ module.exports = function (url, version, dest, cb) {
       }
     });
 
-    src.pipe(gunzip).pipe(tar.Extract({ path: dest }))
-      .on('end',function () {
+    src.pipe(gunzip).pipe(tar.Extract({path: dest}))
+      .on('end', function() {
         console.log(url + ' is now extracted to ' + dest);
-        cb && cb();
-      }).on('error', function (error) {
-        cb && cb(error);
-      });
+        var err;
+        var home = homedir();
+        var icDir = path.join(home, 'oracle-instant-client');
+        if (!fs.existsSync(icDir)) {
+          try {
+            fse.copySync(path.join(dest, 'instantclient'), icDir,
```

Comment:
It looks to me like:
```
stream tar from (disk or http)
extract tar to dst
if icDir not exists:
  copy dst to icDir
```
I'm suggesting:
```
if icDir not exist:
  stream tar from (disk or http)
  extract tar to icDir
```

Ignore me if I'm misreading it, though, of it there's some other reason multiple copies are needed.